system/security/cryptohelper/pke.cpp

```diff
@@ -102,39 +115,69 @@ class CLoadedPublicKeyFromMemory : public CLoadedKey
 
 class CLoadedPrivateKeyFromFile : public CLoadedKey
 {
-public:
-    CLoadedPrivateKeyFromFile(const char *keyFile, const char *passPhrase)
+private:
+    MemoryBuffer passPhraseMB;
```

Comment:
I don't think this needs to be a member of the object.

Perhaps a better implementation would be to change load() to accept a MemoryBuffer& parameter, then change the calling constructors to build a MemoryBuffer as needed.  That would simplify load() as well.  You may also want to consider explicit zeroing the MemoryBuffer contents before destruction.

Same change could be applied to CLoadedPrivateKeyFromMemory.